fraud/paypaldatacollectorutil.md

```diff
@@ -9,6 +9,88 @@ The MobileSDK provides a PayPal Data Collector Util that integrates with the Pay
 
 ## iOS
 
+### 1. Overview
+
+This section provides a step-by-step guide on how to initialize and use the `PayPalDataCollectorUtil` util in your application.
+
+The following sample code demonstrates the definition of the `PayPalDataCollectorUtil`:
+
+```Swift
+public class PayPalDataCollectorUtil {
+    let config: PayPalDataCollectorConfig
+    let clientId: String
+        
+    init(config: PayPalDataCollectorConfig, clientId: String) {
+        self.config = config
+        self.clientId = clientId
+    }
+```
+
+> **Note**:
```

Comment:
@domagoj-grizelj Rather then call out an initialiser they can't use - I wouldn't mention this part. Just the next part - this is how you initialise etc.